docs/package.json

```diff
@@ -29,7 +29,7 @@
     "@types/webpack": "^4.41.26",
     "@types/webpack-dev-server": "^3.11.1",
     "babel-plugin-macros": "^3.0.1",
-    "chroma-js": "^1.3.6",
+    "chroma-js": "^2.4.2",
```

Comment:
I added a Story that was using this package. Figured it doesn't hurt to bump the version to latest.